miner/worker.go

```diff
@@ -369,6 +372,16 @@ func (miner *Miner) makeEnv(parent *types.Header, header *types.Header, coinbase
 }
 
 func (miner *Miner) commitTransaction(env *environment, tx *types.Transaction) error {
+	// OP-Stack addition
+	if len(tx.AccessList()) > 0 {
```

Comment:
Simply having an AccessList does not imply interop, should use `TxToInteropAccessList` instead